### DISCUSSION TITLE
Open the second step when open joyride,i want to not show the first step like "lets begin our journey"  in the demo page

### DISCUSSION THREAD

**Discussion by hieule2k** [gilbarbara/react-joyride#977] [Q&A] [answered]
I dont want to open the first step at center 
there is anyway that i can open it when my page is first loaded but with the specific location intend 
i want to see this on the first step
![image](https://github.com/gilbarbara/react-joyride/assets/91259565/6a96feb0-98f6-4c63-922f-32e1c17ae758)

I dont want to show this on the first step
![image](https://github.com/gilbarbara/react-joyride/assets/91259565/3883aaa4-f406-417f-9f5f-6f08c7f0bc5a)

**Comment by gilbarbara** [comment, marked as answer]
Hey @hieule2k 
This isn't an issue with the library, just an implementation question, so I moved it to discussion.

Yes, you can control the `run` prop in your app to change when your requirements are met. 
About the steps, sure, you can set any step you want.
